1223-curator_worklist/my_worklist.feature

```diff
@@ -115,71 +141,36 @@ Feature: Curator opens My Worklist
       | Archived |
       | Deleted  |
 
   @needJiraTag
-  Scenario: Curator claims a Request
-    When the Curator do either:
-      | send an answer        |
-      | Publish               |
-      | Mint a DOI            |
-      | Decline a DOI         |
-      | Change Owner          |
-      | Decline a Ownerchange |
-    Then the Curator is stored as Owner of the Request
-
-  @needJiraTag
-  Scenario: Decide which Approval, Support or DOI Requests is in a Curators Scope
-    When the Requests' Host equals the Curators Host
-    And Curators scope on the Institution include the Requests' Users' User Profile affilliation to the Institution
-    Then the Users Request is displayed to the Curator
-
-
-  @needJiraTag
-  Scenario: Curator opens their Work List
-    When the Curator opens the Work List
-    And it is scoped to an institutional subunit
-    Then the Curator see it is scoped to a subunit
-    And all Requests from Senders in this subunit is listed
-
+  Scenario: Curator is Assigned a Request
+    When the Curator:
+      | sends an answer          |
+      | Publishes a resource     |
+      | Mints a DOI              |
+      | Declines a DOI           |
+      | Changes Owner            |
+      | Declines change of owner |
+    Then the Curator is Assigned the Request
 
   @needJiraTag
-  Scenario: Curator changes scope of Work List
-    When the Curator opens the Work List
-    Then the Curator may change scope to any other subunit at the institution
-    And all Requests from Senders in this new subunit is listed
+  Scenario: Decide if an Approval, Support or DOI Requests is in a Curators Scope
+    Given the Request is of type:
+      | Approval |
+      | Support  |
+      | DOI      |
+    When the Requests' Submitter is Affilliated within the Curators Scope
+    And regardless of Scope, if the Curator is assigned the Request
+    Then the Request is part of the Curators Worklist
 
   @needJiraTag
-  Scenario: Curator change scope of there Work List to Institution level
-    When the Curator opens the Work List
-    Then the Curator may change scope to the whole Institution
-    And all Requests from all Senders is listed
-
-  Scenario: Curator performs action on Request
-    When the Curator opens the Work List
-    And relevant Requests from Senders is listed
-    Then the Curator may perform actions on listed Requests
-
-
-  @needJiraTag
-  Scenario: Curator performs action on Request
-    When the Curator opens the Work List
-    And relevant Requests from Senders is listed
-    Then the Curator may perform actions on listed Requests
-
-
-  Scenario: Decide which Ownership Requests is in a Curators Scope
-    When the Requests' Resources' Host equals the Curators Host
-    And Curators scope on the Institution include the Resources' Owners' User Profile affilliation to the Institution
-    Then the Users Request is displayed to the Curator
-
-
-
-
-
-
+  Scenario: Decide if an Ownership Requests is in a Curators Scope
```

Comment:
Curator receives Ownership requests within their scope (but not ones they have previously responded to?)

Reply:
Duplicated the scenario and adjusted